content/en/blog/_posts/2021-22-02-The-Evolution-of-Kubernetes-Dashboard/index.md

```diff
@@ -0,0 +1,69 @@
+
+The very [first commit](https://github.com/kubernetes/dashboard/commit/5861187fa807ac1cc2d9b2ac786afeced065076c) to the Kubernetes Dashboard was made by Filip Grządkowski from Google on 16th October 2015 – just a few months from the initial commit to the Kubernetes repository. Our initial commits go back to November 2015 ([Sebastian committed on 16 November 2015](https://github.com/kubernetes/dashboard/commit/09e65b6bb08c49b926253de3621a73da05e400fd); [Marcin committed on 23 November 2015](https://github.com/kubernetes/dashboard/commit/1da4b1c25ef040818072c734f71333f9b4733f55)). Since that time, we’ve become regular contributors to the project. For the next two years, we will be working closely with the Googlers, eventually becoming main project maintainers ourselves.
+
+{{< imgproc first-ui Fit "1000x900" >}}
+The First Version of the User Interface
+{{< /imgproc >}}
+
+{{< imgproc along-the-way-ui Fit "1000x900" >}}
+Prototype of the New User Interface
+{{< /imgproc >}}
+
+{{< imgproc current-ui Fit "1000x700" >}}
+The Current User Interface
+{{< /imgproc >}}
+
+As you can see, the initial look and feel of the project were completely different from the current one. We have changed the design multiple times. The same has happened with the code itself.
+
+## Growing Up - The Big Migration
+
+At [the beginning of 2018](https://github.com/kubernetes/dashboard/pull/2727), we reached a point where AngularJS was getting closer to the end of its life, while the new Angular versions were published quite often. A lot of the libraries and the modules that we were using were following the trend. That forced us to spend a lot of the time rewriting the frontend part of the project to make it work with newer technologies.
+
+The migration came with many benefits like being able to refactor a lot of the code, introduce design patterns, reduce code complexity, and benefit from the new modules. However, you can imagine that the scale of the migration was huge. Luckily, there were a number of contributions from the community helping us with the resource support, new Kubernetes version support, i18n, and some other things. After many long days and nights, we finally released the [first beta version](https://github.com/kubernetes/dashboard/releases/tag/v2.0.0-beta1) in July 2019, followed by the [2.0 release](https://github.com/kubernetes/dashboard/releases/tag/v2.0.0) in April 2020 — our baby had grown up.
+
+## Where Are We Standing in 2021?
+
+Due to limited resources, unfortunately, we were not able to offer extensive support for many different Kubernetes versions. So, we’ve decided to always try and support the latest Kubernetes version available at the time of the Kubernetes Dashboard release. The latest release, [Dashboard v2.2.0](https://github.com/kubernetes/dashboard/releases/tag/v2.2.0) provides support for Kubernetes v1.20.
```

Comment:
I have updated latest release info to `v2.2.0` as it was released recently.

Reply:
How does it work with the date of the post?

Reply:
Date of the release 18.02
Date of the blog post 22.02

Reply:
Once this is ready for review we can push a commit to match the date it'll go live.

We can accept future-dated articles but past dated ones I think we prefer not to (there's no CI test or anything, it's just odd for readers when an article turns up today that says it was published last week).